src/plugins/python2/CMakeLists.txt

```diff
@@ -53,7 +53,7 @@ endif ()
 if (ADDTESTING_PHASE)
 	# bindings are required for tests
 	list (FIND BINDINGS "swig_python2" FINDEX)
-	if (NOT BUILD_SHARED AND BUILD_TESTING AND FINDEX GREATER -1)
+	if (BUILD_TESTING AND FINDEX GREATER -1)
```

Comment:
This update causes almost the same problems, as the update of the Python 3 plugin. I think we should keep the tests disabled for now, until someone fixes them.

Reply:
Then we should only disable them for Apple, for Linux the tests seem to work.

But we should document which tests are disabled (for which platform).

Reply:
> Then we should only disable them for Apple, for Linux the tests seem to work.

Then we still need to fix the build job [`elektra-gcc-configure-debian-log`](https://build.libelektra.org/jenkins/job/elektra-gcc-configure-debian-log/971/console), since the tests of both Python plugins cause a segmentation fault there.